README.md

```diff
@@ -19,4 +21,14 @@ A VS Code extension for Open Liberty. The extension will detect your Liberty Mav
 | liberty.terminal.useJavaHome | If this value is true, and if the setting `java.home` has a value, then the environment variable `JAVA_HOME` will be set to the value of `java.home` when a new terminal window is created. | False |
 
 ## Contributing
-Our [CONTRIBUTING](CONTRIBUTING.md) document contains details for submitting pull requests.
+Contributions to the Open Liberty Tools extension are welcome! Our [CONTRIBUTING](CONTRIBUTING.md) document contains details for submitting pull requests.
+
+To build the extension locally:
+1. `git clone git@github.com:OpenLiberty/liberty-dev-vscode-ext.git`
```

Comment:
This wouldn't work if the user hasn't set up an SSH key.  It can be more generic to specify https e.g. `git clone https://github.com/OpenLiberty/liberty-dev-vscode-ext`